### ISSUE TITLE
[Feature] publication collaborators

### ISSUE THREAD

**Issue by chory** [e107inc/e107#3916]
Allow the creator of a publication, indicate that collaborators can modify it.
This feature would be when a user is going to be absent, to be able to select other users, to keep an updated publication, or so that several users can keep it updated.

**Comment by Moc** [comment]
Sounds like an interesting idea. Are your refering to the pages/menus functionality?

**Comment by chory** [comment]
I refer to the forums mainly, but it could be applied to other sections.
For example, if it is a download website, the creator of a publication of a television series, at a given time, cannot continue with the publication, it could delegate to another user to continue with the task.

**Comment by Moc** [comment]
Hmm this may be trickier than expected, and also different for various sections. Ideally, you'd use userclasses or userclass groups for all of the sections. 

For the download plugin, you could give someone admin access to the download plugin specifically but then that person would be able to edit all downloads. 

For the forum you'd be dealing with forum categories vs individual forum topics. 

Would like to hear what other people think about this.

**Comment by chory** [comment]
For the forum it could be used as for books / chapters, with the field 'Can be edited by' for books / chapters, the idea would be to have this feature in the main thread.

**Comment by Jimmi08** [comment]
Isn't easier to change creator to someone new? 
Classes are the correct solution.  And the forum already has moderators.   
Sorry, I don't understand fully what is the goal.

**Comment by chory** [comment]
> Isn't easier to change creator to someone new?
> Classes are the correct solution. And the forum already has moderators.
> Sorry, I don't understand fully what is the goal.

The goal is for any user to decide that another user can help them maintain / update a topic in a forum, without the need for that user to be an administrator / moderator on the web.
Because he is absent, or simply because he cannot update it personally.
I am making a website, to offer information about my city.
My intention is to make a section so that the different cultural / sports / recreational associations and businesses can offer information to the rest of the citizens regarding the activities they program or carry out, or the prices and offers of their products.
What I want is to be able to designate a user member of each association, so he can publish that information. So far it is easy, but I want this user to define himself, that another or other users can modify that information, without having to modify the privileges of website users, or creating new user classes.
Imagine that an association organizes a trip.
The person indicated initially (the president of the association), makes it public for people to find out.
But soon after, she is sick, and cannot update that information.
What I propose is that this person has previously been able to delegate to another user, so that if something like this happened, he could update the information, without having to provide his access data.

**Comment by Jimmi08** [comment]
Thank you, good luck with this.  I don't know a forum system that allows this.
PS. In my former work, we used for this user system based on roles. So there was a special user 'association_manager_01' _02 etc, which post and edit that information for one association. If they quick (be ill), the password was changed and access was given to another user.  So one user has access to 2 users account. His own with all his personal information and user for managing published information (and it used company email because it was created by company).

**Comment by chory** [comment]
Yes, something like that I had done on another website, and it seems a bit "chaotic" to have to modify privileges, for something specific.
I find it more "simple", less complicated, and a way of not burdening administrators with work, that one user can delegate to another to do a task.
After all, it is as it is done in real life among people, "I designate You to help me with this task, or replace me when I am not there."
